### PR TITLE
fix: timeout in runner/kubectl operations

### PR DESCRIPTION
**Pull request by shubham-cmyk**
Signed-off-by: Shubham Gupta <iamshubhamgupta2001@gmail.com>

## Related issue

Fixes https://github.com/kyverno/chainsaw/issues/929

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
- [x] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
- [ ] This is a bug fix and I have added unit tests that prove my fix is effective.

### CODE REVIEW

File: pkg/runner/kubectl/describe_test.go [hunk 1]
```diff
@@ -14,133 +14,158 @@ func TestDescribe(t *testing.T) {
...
-		collector: &v1alpha1.Describe{
-			ObjectLabelsSelector: v1alpha1.ObjectLabelsSelector{
-				Name: "foo",
+	}{
+		{
+			name:      "nil",
+			collector: nil,
+			want:      nil,
+			wantErr:   false,
+		},
```
Comment (eddycharly) [inline pkg/runner/kubectl/describe_test.go:23]: why did you change the indentation ?

Comment (shubham-cmyk) [reply pkg/runner/kubectl/describe_test.go:23]: it changed automatically

Comment (eddycharly) [reply pkg/runner/kubectl/describe_test.go:23]: would be great ! the old indentation is more compact :)

Comment (shubham-cmyk) [reply pkg/runner/kubectl/describe_test.go:23]: okay reverting back

**Comment by eddycharly** [comment]
@shubham-cmyk you can simply do
```golang
	cmd := v1alpha1.Command{
                Timeout: collector.Timeout,
                // ...
	}
```

**Comment by shubham-cmyk** [comment]
> @shubham-cmyk you can simply do
> 
> ```go
> 	cmd := v1alpha1.Command{
>                 Timeout: collector.Timeout,
>                 // ...
> 	}
> ```

done

File: pkg/runner/kubectl/describe_test.go [hunk 2]
```diff
@@ -81,6 +81,17 @@ func TestDescribe(t *testing.T) {
 			Args:       []string{"describe", "foos", "foo", "-n", "bar"},
 		},
 		wantErr: false,
+	}, {
```
Comment (eddycharly) [inline pkg/runner/kubectl/describe_test.go:84]: this test is a duplicate

**Review by eddycharly — APPROVED** [review]
